src/test/functions/projectConfig.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { expect } from "chai";
+
+import * as projectConfig from "../../functions/projectConfig";
+import { FirebaseError } from "../../error";
+
+const TEST_CONFIG = { source: "foo" };
+
+describe("projectConfig", () => {
+  describe("normalize", () => {
```

Comment:
What if it's given more than one? Should that have a test?

Reply:
That is checked in `validate`, and we have a test case for that.